deployment/pypi/README.md

```diff
@@ -53,9 +53,10 @@ This is the PyPI build and upload tool for NNI project.
   ```
 
 * __How to build__
-
+ 
+  parameter `version_os` is used to choose Windows version 64-bit or 32-bit, `version_ts` is used to build for test or not. 
   ```bash
-  powershell ./install.ps1
+  powershell ./install.ps1 -version_os [64/32] -version_ts [$True/$False]
```

Comment:
Can you remove the -version_ts option from doc?   it could be change or removed from code.

Reply:
I can not understand the change. Could you give more details?

Reply:
-version_ts option is an implementation detail,  should not be mentioned in document.

Reply:
It decides whether to build pypi package for test or release. Should we remove the parameters -version_os and -version_ts?